doc/ref_model/chapters/chapter04.md

```diff
@@ -460,8 +460,8 @@ N instance types can come with Network Acceleration extensions to assist VNFs of
 
 | .conf | Interface type | Description |
 |------------|----------------|-----------------------------------------|
-| .il-ipsec | virtio-ipsec* | In-line IPSec acceleration |
-| .la-crypto | virtio-crypto | Look-Aside encryption/decryption engine |
+| .il-ipsec | virtio-ipsec* | In-line IPSec acceleration. <br> **Note:** In Kubernetes based infrastructures virtio-ipsec can be applied only if the CNI plugin is OVS. |
+| .la-crypto | virtio-crypto | Look-Aside encryption/decryption engine. <br> **Note:** In Kubernetes based infrastructures virtio-crypto can be applied only if the CNI plugin is OVS. |
```

Comment:
@CsatariGergely RA compliance matrix?

Reply:
I still need to familiarize myself with the RA compliance matrix.

Reply:
#738 was created